include/deal.II/particles/particle_handler.h

```diff
@@ -51,9 +51,20 @@ namespace Particles
    * is designed in a similar way as the triangulation class. In particular,
    * we call particles in the domain of the local process local particles,
    * and particles that belong to neighbor processes and live in the ghost cells
-   * around the locally owned domain "ghost particles".
+   * around the locally owned domain "ghost particles". The class also includes
+   * functionality that is similar to the DoFHandler() class (it knows which
+   * particles live on which cells) and the SolutionTransfer() class (it know
+   * how to transfer particles between cells and subdomains).
    *
-   * This class is used in step-70.
+   * @note: While the class can be used in any kind of triangulation, transfer
```

Comment:
```suggestion
   * @note While the class can be used in any kind of triangulation, transfer
```

Reply:
We're likely going to forget about this note when the whole thing is finally fixed. My preference would be to put the whole note into an assertion that errors out if someone tries this. We could attach it to one of the signals.

Reply:
That assertion already exists as part of #12450. I removed the note.